src/ldf-client-ui.js

```diff
@@ -4,7 +4,11 @@
 // This exports the webpacked jQuery.
 window.jQuery = require('../deps/jquery-2.1.0.js');
 var N3 = require('n3');
-
+// TODO: Dynamically expose the version in ActorInitSparql so that we can retrieve it from there instead of relying on package.json reading.
+window.jQuery(function ($) {
```

Comment:
On line 16, an explicit jQuery context is defined, so that jQuery is present via the `$` constant.
So I would move your two lines in that context. Preferably completely at the end, preceded by an empty line.